pkg/qemu/qemu.go

```diff
@@ -61,14 +62,19 @@ func EnsureDisk(cfg Config) error {
 		}
 	}
 	diskSize, err := units.RAMInBytes(cfg.LimaYAML.Disk)
+	if err != nil || diskSize == 0 {
+		return err
```

Comment:
return a new error when `diskSize == 0 && err == nil`(does it happen? it should be already validated)

Reply:
I think `err` will always be `nil` at this point, so the logic really is:

```go
	diskSize, _ := units.RAMInBytes(cfg.LimaYAML.Disk)
	if diskSize == 0 {
		return nil
        }
```

Maybe it is clearer, but then I would always wonder why it is safe to ignore the error... 😄

Reply:
Confirmed that this call cannot fail, so ignoring error return value